src/utils/api.js

```diff
@@ -94,7 +94,10 @@ const callAPI = (route, options) => {
 
       if (constants.DEBUG || global.argOpts.debug) {
         console.log(`>> ${requestOptions.method} ${requestOptions.url}`);
-        if (requestOptions.body) { console.log(`>> ${requestOptions.body}`); }
+        if (requestOptions.body) {
+          let cleanedBody = _.assign({}, JSON.parse(requestOptions.body), {zip_file: 'raw zip removed in logs'});
```

Comment:
Please, I don't get how the raw zip is removed. Is it an item with key `zip_file` in the outcome of `JSON.parse(requestOptions.body)`? If so, I got it then.